app/models/user.rb

```diff
@@ -3,7 +3,7 @@ class User < ApplicationRecord
   has_many :videos, through: :user_videos
 
   validates :email, uniqueness: true, presence: true
-  validates :password, presence: true
+  validates :password_digest, presence: true
```

Comment:
password_digest to fix error 'password cannot be blank'

Reply:
Nice catch, would not have thought to change that.